app/scripts/proactive/templates/job-template.html

```diff
@@ -44,4 +44,13 @@
                 <%=task%>
         <% }) %>
     </taskFlow>
+
+    <metadata>
+        <visualization>
+            <% if (visualization) { %>

```

Comment:
Is visualization created only when you submit a workflow or export to the catalog ?
If yes, I'm afraid we will miss the visualization when we simply export the workflow to XML, or that the visualization may be wrong.

Could you double-check this ?

Reply:
No, visualization is updated each time the workflow is saved